tox.ini

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
`flake8` is a command to verify pep8 compatibility of all source files. I've added rules so it would allow longer lines in this project as we both use bigger monitors.